packages/mux-player/src/helpers.ts

```diff
@@ -80,9 +80,10 @@ export function getCcSubTracks(el: MuxPlayerElement) {
 
 export const getLiveTime = (el: MuxPlayerElement) => {
   const { media } = el;
-  return media?._hls?.liveSyncPosition ?? media?.seekable.length
-    ? media?.seekable.end(media.seekable.length - 1)
-    : undefined;
+  return (
+    media?._hls?.liveSyncPosition ??
+    (media?.seekable.length ? media?.seekable.end(media.seekable.length - 1) : undefined)
+  );
```

Comment:
cc @cjpillsbury was a bug where `media?.seekable.end(media.seekable.length - 1)` was always tried to get because of condition priority. resulted in this error sometimes
![SCR-20220602-qqa](https://user-images.githubusercontent.com/360826/171912428-2d943199-ce36-432b-8b04-dc30b05218cd.jpeg)